backend/src/clients/kms.ts

```diff
@@ -0,0 +1,42 @@
+import { DescribeKeyCommand, KMSClient, SignCommand, SignCommandOutput } from '@aws-sdk/client-kms'
+import { ASN1HEX } from 'jsrsasign'
+
+import config from '../utils/config.js'
+import { InternalError } from '../utils/error.js'
+
+const uint8ArrayFromHexString = (hexstring) =>
```

Comment:
Is there a reason you're doing regular expression matching over a simpler `Buffer.from(hexstring, 'hex')` here?